src/lib/output/renderer.ts

```diff
@@ -206,14 +206,12 @@ export class Renderer extends ChildableComponent<Application, RendererComponent>
                 this.theme = this.addComponent('theme', new DefaultTheme(this, path));
             } else {
                 try {
-                    const themeClass = typeof require(filename) === 'function' ? require(filename) : require(filename).default;
+                    const themeClass = typeof require(filename) === 'function' ? require(filename)() : require(filename).default;
```

Comment:
I'm not sure I understand - why's it such a big deal to share why you want this change? There shouldn't be any difference between the two export styles, so I'm trying to understand where your difference is coming from.